app/components/dex-create.jsx

```diff
@@ -7,14 +7,29 @@ import slug          from 'slug';
 import { AlertComponent } from './alert';
 import { ReactGA }        from '../utils/analytics';
 import { createDex }      from '../actions/dex';
+import { listGames }      from '../actions/games';
 
 const NATIONAL_ONLY_GAMES = ['x', 'y', 'omega_ruby', 'alpha_sapphire'];
 
 export class DexCreate extends Component {
 
   constructor (props) {
     super(props);
-    this.state = { error: null, game: 'sun', regional: false };
+    this.state = { error: null, game: 'ultra_moon', games: [], regional: false };
```

Comment:
we wont be saving `games` in the local component state (i.e. `this.state = {}` and `this.setState({})`). instead, the action will be saving it in the global state. and instead of pulling it from `this.state`, we'll be pulling it from `this.props` because `mapStateToProps` will pull it form the global state and set it as the components `props`